static/_redirects

```diff
@@ -268,3 +268,4 @@
 /capacity-monitoring/metrics-page/ https://docs.virtana.com/en/metrics.html 301!
 /data-visualization/metrics https://docs.virtana.com/en/metrics.html 301!
 /alerts-notificaitons/policies https://docs.virtana.com/en/policies.html 301!
+/docs.metricly.com/ https://docs.virtana.com/ 301!
```

Comment:
Shouldn't it be this?

```suggestion
/ https://docs.virtana.com/ 301!
```

Reply:
entire Preview site is redirected to https://docs.virtana.com/?lang=en